### PR TITLE
[WIP] Fixing the bot

### PR DESCRIPTION
**Pull request by scarlehoff**
Given that we have a case that is supposed to fail, let's see whether we can actually reproduce it.

### CODE REVIEW

**Comment by Zaharid** [comment]
I wonder if I can do something to make the error message better for that problem.

**Comment by scarlehoff** [comment]
Yes, something like "it seems we are in a period of high cosmic activity"

File: .github/workflows/fitbot.yml [hunk 1]
```diff
@@ -114,7 +114,7 @@ jobs:
       if: failure()
       shell: bash -l ${0}
       run: |
-        scp /usr/share/miniconda/envs/test/conda-bld/linux-64/nnpdf*.tar.bz2  nnpdf.fisica.unimi.it:debug_pkgs/
+        scp -o StrictHostKeyChecking=no $CONDA_PREFIX/conda-bld/${{matrix.CONDA_OS}}/*.tar.bz2 nnpdf@fisica.unimi.it:debug_pkgs/
```
Comment (scarlehoff) [inline .github/workflows/fitbot.yml:117]:
```suggestion
        ls $CONDA_PREFIX/conda-bld/*/*.tar.bz2
        scp -v -o StrictHostKeyChecking=no $CONDA_PREFIX/conda-bld/*/*.tar.bz2 nnpdf@fisica.unimi.it:debug_pkgs/
```

**Comment by scarlehoff** [comment]
Well, it consistently fails even with master so I should be able to reproduce that in my computer if I do enough loops.

**Pull request closed without merging**